src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -276,7 +281,7 @@ public boolean clickShowNewInstructorFormButton() {
     
     public boolean clickOnAccessLevelViewDetails(String role) {
         WebElement viewDetailsLink = browser.driver.findElement(By.cssSelector(
-                                            "a[onclick=\"showInstructorRoleModal(\'" + role + "\')\"]"));
+                                            "a[onclick=\"showInstructorRoleModal('" + role + "')\"]"));
```

Comment:
Not possible, the selector becomes illegal due to it interpreting it as `a[onclick='showInstructorRoleModal'` followed by an unexpected string.